main.js

```diff
@@ -494,7 +494,7 @@ function parseIniFromText(text) {
 
 function flowey_laugh_once() {
     "use strict";
-    if (localStorage.getItem("laughed") !== "true") {
+    if (localStorage.getItem("laughed") !== "true" && document.getElementById("volume").checked) {
```

Comment:
I just noticed, this prevents the image from getting changed if the volume is muted. Should I move the checkbox check below to only include lines that are playing audio?